Matteo Bernardini <ponce@slackbuilds.org>
- - -
Fix for the newer cmake routine to find boost_python libraries,
see /usr/share/cmake-3.11/Modules/FindBoost.cmake

diff -Naur FreeCAD-0.17.orig/src/Mod/Path/libarea/CMakeLists.txt FreeCAD-0.17/src/Mod/Path/libarea/CMakeLists.txt
--- FreeCAD-0.17.orig/src/Mod/Path/libarea/CMakeLists.txt	2018-04-06 10:48:37.000000000 +0200
+++ FreeCAD-0.17/src/Mod/Path/libarea/CMakeLists.txt	2018-05-08 12:32:38.690601000 +0200
@@ -17,12 +17,12 @@
 if(USE_BOOST_PYTHON)
     if(NOT FREECAD_LIBPACK_USE OR FREECAD_LIBPACK_CHECKFILE_CLBUNDLER)
         if(NOT PYTHON_VERSION_MAJOR LESS 3)
-            find_package( Boost COMPONENTS python3)
+            find_package( Boost COMPONENTS python36)
             if (NOT Boost_PYTHON3_FOUND)
-                find_package( Boost COMPONENTS python REQUIRED)
+                find_package( Boost COMPONENTS python27 REQUIRED)
             endif()
         else()
-            find_package( Boost COMPONENTS python REQUIRED)  # find BOOST and boost-python
+            find_package( Boost COMPONENTS python27 REQUIRED)  # find BOOST and boost-python
         endif()
 
         if(Boost_FOUND)
